pipeline/pipelines/gapic_generation_pipeline.py

```diff
@@ -52,20 +51,26 @@ def __init__(self, **kwargs):
             **kwargs)
 
 
+class GapicClientBatchPipeline(batch_gen.BatchPipeline):
+
+    def get_pipeline_tasks(self, **kwargs):
+        task_factory = get_gapic_task_factory(kwargs['language'])
+        return task_factory._get_gapic_codegen_tasks(**kwargs)
```

Comment:
I think we need to do the following:

1. Separate the pipelines for codegen from packaging (not just separate them for the batch case). 
2. Decouple the command line from the implementing classes inside artman; we should come up with identifiers to use, which would determine which pipeline(s) would be created. 

This might be a good surface:

```
# perform codegen for one language/API combo
artman gapic-codegen --language java --api pubsub

# batch APIs, one language
artman gapic-codegen --language java --api all

# batch languages, one API
artman gapic-codegen --language all --api pubsub

# perform packaging
artman gapic-package --language java --api pubsub

# perform codegen + packaging
artman gapic-codegen gapic-package --language java --api pubsub

# perform batch codegen + packaging for all languages and APIs
artman gapic-codegen gapic-package --language all --api all

```